recipe/meta.yaml

```
@@ -1,56 +1,63 @@
-{% set version = "1.6.7" %}
+{% set version = "1.7.1" %}
 
 package:
   name: dbt
   version: {{ version }}
 
 source:
   - url: https://pypi.io/packages/source/d/dbt-core/dbt-core-{{ version }}.tar.gz
-    sha256: 09c76b6b9bde3f8e00d24451a18af5b1c8151abfda9884603ce9d5bafc24c556
+    sha256: 78ac4711f35f7b7b1ad4bb17181b900ff9e15aed424e432e336fe110fec5981e
     folder: dbt-core
   - url: https://pypi.io/packages/source/d/dbt-postgres/dbt-postgres-{{ version }}.tar.gz
-    sha256: e6109322084316116d44717e57fb249edad8cf8ee05e83840a986c825cae07a9
+    sha256: cef8ffcf79d352d8cbc89a939753a546f2aad183688bd78743f385882bb9e001
     folder: dbt-postgres
+  - url: https://github.com/dbt-labs/dbt-core/archive/refs/tags/v{{ version }}.zip
+    sha256: 226f508d23edd9ad7688db68829c0a10b19f33ac79ceecd03474b467fe2839de
+    folder: dbt-core-git
 
 build:
   number: 0
-  noarch: python
+  # Skip s390x because we don't have agate on s390x.
+  skip: true  # [py<38 or s390x]
 
 outputs:
   - name: dbt-core
     build:
-      noarch: python
       entry_points:
         - dbt = dbt.cli.main:cli
-      script: python -m pip install --no-deps  -vv ./dbt-core
+      script: python -m pip install --no-deps --no-build-isolation -v ./dbt-core
     requirements:
       host:
         - pip
-        - python >=3.8
+        - python
+        - setuptools
+        - wheel
       run:
-        - python >=3.8
+        - python
 
         # Mashumaro is installed as "mashumaro[msgpack]", so the following comes from
         # the "extras_require" section of https://github.com/Fatal1ty/mashumaro/blob/master/setup.py
         - msgpack-python >=0.5.6
 
         # Generated with Grayskull
-        - agate >=1.7.0,<1.8.dev0
+        - agate >=1.7.1,<1.8.dev0
         - jinja2 >=3.1.2,<3.2.dev0
-        - mashumaro >=3.8.1,<3.9.dev0
+        - mashumaro >=3.9
         - logbook >=1.5,<1.6
-        - click <9
+        - click >=8.0.2,<9
         - networkx >=2.3,<4
         - colorama >=0.3.9,<0.5
         - pathspec >=0.9,<0.12
         - isodate >=0.6,<0.7
         - sqlparse >=0.2.3,<0.5
-        - dbt-extractor >=0.4.1,<0.5.dev0
+        - dbt-extractor >=0.4.0,<0.6.dev0
```

Any reason for not setting this:
```suggestion
        - dbt-extractor >=0.5.0,<0.6.dev0
```
https://github.com/dbt-labs/dbt-core/blob/v1.7.1/core/setup.py#L74
It's difficult to tell from looking at this changelog. https://github.com/dbt-labs/dbt-extractor/blob/main/CHANGELOG.md

---

No reason. It's probably a mistake from me. Good catch!